### PR TITLE
Mppt packet digital

### PR DESCRIPTION
**Pull request by magicrub**
two new packets for MPPT monitor by Packet Digital. This PR is needed for https://github.com/ArduPilot/ardupilot/pull/19404

### CODE REVIEW

File: mppt/240.OutputEnable.uavcan [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#
+# Service to enable or disable the mppt output
+#
+
+void6
+
+# If both enable and disable are true then no change is applied
+bool enable
```
Comment (tridge) [inline mppt/240.OutputEnable.uavcan:8]: why have two bool values? why not just the enable value, and set to false to dissable?

**Review by tridge — CHANGES_REQUESTED** [review]

File: mppt/240.OutputEnable.uavcan (hunk 1 above)
Comment (tridge) [reply mppt/240.OutputEnable.uavcan:8]: if it is needed, then a comment explaining why we have both would be nice

Comment (magicrub) [reply mppt/240.OutputEnable.uavcan:8]: This was given to me by the manufacturer. I didn't have any input on it's design. I totally agree that it's a less than ideal method. Meanwhile, the company has shipped hardware using this.

Comment (magicrub) [reply mppt/240.OutputEnable.uavcan:19]: I've re-worked this. It's much more verbose now

**Review by tridge — APPROVED** [review]

**Pull request merged**